Headers/CoreGraphics/CGContext.h

```diff
@@ -138,6 +138,22 @@ enum {
 };
 typedef int CGTextEncoding;
 
+// NOTE: This `typedef enum` in opposition to defining enum constants and
```

Comment:
Where did you find these values?  As WebKit has its own definitions of these values we should kept them private within Opal.